README.md

```diff
@@ -3,8 +3,10 @@
 Seal
 ====
 
-Performs sealing of Red Hat Enterprise Linux machine described at
-[oVirt site](http://www.ovirt.org/documentation/how-to/virtual-machines/sealing-linux-vm/).
+This role performs sealing steps for following operating systems:
+
+* RedHat / CentOS  ([oVirt site](http://www.ovirt.org/documentation/how-to/virtual-machines/sealing-linux-vm/).)
```

Comment:
just nit: you left dot (.) at the end of link ...

when you will update this you can also move link on the bottom of file and use only `[oVirt Site]` here to make it more human readable ;-)